pkg/tests/service/options.go

```diff
@@ -345,19 +347,25 @@ func (opt Options) WithKeepData() Options {
 // then select all of them.
 func gossipSeedNum(logServiceNum int) int {
 	if logServiceNum < defaultGossipSeedNum {
-		return logServiceNum
+		return defaultGossipSeedNum
 	}
-	return defaultGossipSeedNum
+	if logServiceNum > maxGossipSeedNum {
+		return maxGossipSeedNum
+	}
+	return logServiceNum
 }
 
 // haKeeperNum calculates the count of hakeeper replica.
 //
 // Select the first 3 log services to start hakeeper replica.
 // If the number of log services was less than 3,
 // then select the first of them.
```

Comment:
Please sync this comment with the new logic.

Reply:
updated